docs/MSPX.md

```diff
@@ -3,14 +3,16 @@
 ([back to main page](../README.md))
 
 mLRS provides the MspX technology, which is designed to improve the over-the-air communication for systems using the MSP protocol, specifically INAV. It includes the following features:
-- MSP to CRSF message conversion which provides telemetry elements to the radio and therefore enables Lua scripts on the radio such as the INAV app or telemetry widgets to function.
+- MSP to CRSF message conversion which provides telemetry elements to the radio and therefore enables Lua scripts on the radio such as the INAV Telemetry Widget, FM2M Toolbox or the Horus Mapping Widget to function.
 - Robust framing and parsing which reduces packet losses to a minimum, and compression of some very large MSP messages to increase probability of successful transmission.
+- Providing of comprehensive mLRS link statistics and information to the Flight-Controller, to show them on the OSD and in blackbox recordings.
+- Possibility of RC-Control, serial data communication and Radio Telemetry through a single UART. 
 
 ***Notes***: 
-- MspX is not strictly compliant with the MSP protocol, and softwares which require strict compliance may not work properly. This includes e.g. MWP. In these cases, mLRS can be set into transparent mode by setting "Rx Ser Link Mode" = "transp." (the below features of MspX will then not be avaiable).
-- MspX is tested and verified for INAV 7.1. It has been successfully tested with INAV 8.0 development builds (it may also work with INAV 6.0 and newer, but this has not been tested).
+- For INAV Versions before 8 RC1 MspX is not strictly compliant with the MSP protocol, and softwares which require strict compliance may not work properly. This includes e.g. MWPT. In these cases, mLRS can be set into transparent mode by setting "Rx Ser Link Mode" = "transp." (the below features of MspX will then not be avaiable).
```

Comment:
Perhaps

'It is strongly recommended to use INAV 8 or newer, as earlier versions may not work properly due to updates in the MSP protocol.' 

I also think the sentence below 'MspX is tested and verified for INAV 7.1 and newer (it may also work with older INAV releases, but this has not been tested).' can be deleted.

Reply:
I have shortened and rephrased it. The main caviat is, that 7.1 works flawless with MspX and mLRS as a radio and telemetry link. The main limitation is only the GCS connection issues and when used with MspX+CRSF its about as good for RC and Telemetry only as 8.0. 
But I reduced the details a bit and recommended INAV 8. 

Hope thats okay.